enigma-principal/app/src/boot_network/principal_manager.rs

```diff
@@ -212,6 +212,7 @@ impl Sampler for PrincipalManager {
     ///
     /// # Arguments
     ///
+    /// * `path` - path to the .enigma directory.
```

Comment:
Does this parameter always have the same value? If so , then this parameter is redundant.